x/bridge/keeper/keeper.go

```diff
@@ -23,9 +23,9 @@ type Keeper struct {
 	router *baseapp.MsgServiceRouter
 	// accountKeeper helps get the module's address
 	accountKeeper types.AccountKeeper
-	// govModuleAddr is used in UpdateParams method since it is
+	// authority is used in UpdateParams method. It is
 	// the only addr that can update bridge module params
-	govModuleAddr string
+	authority string
```

Comment:
IMO we should keep this as govModuleAddr, there should be no case where this is anything else